theforeman.org/pipelines/lib/release.groovy

```diff
@@ -1,6 +1,19 @@
-void push_foreman_rpms(repo_type, version, distro) {
+void push_foreman_rpms(repo_type, version, distros) {
     version = version == 'develop' ? 'nightly' : version
-    push_rpms("foreman-${repo_type}-${version}", repo_type, version, distro)
+    keep_old_files = version != 'nightly'
+
+    for (distro in distros) {
+        if repo_type {
+            // Main foreman repository
+            if version == 'nightly' {
+                push_rpms_direct("foreman-${foreman_version}/${release}", "${foreman_version}/${release}")
+            } else {
+                push_rpms("foreman-${version}", 'releases', version, distro, keep_old_files)
+            }
+        } else {
+            push_rpms("foreman-${repo_type}-${version}", repo_type, version, distro, keep_old_files)
```

Comment:
If it makes it this far, I would think `repo_type` will be null/false and this line won't make as much sense?

Reply:
See, there was a mistake. This should be the other way around: here repo_type must be defined. Will update.